src/agent/Cargo.toml

```diff
@@ -11,7 +11,7 @@ rustjail = { path = "rustjail" }
 protocols = { path = "protocols" }
 netlink = { path = "netlink", features = ["with-log", "with-agent-handler"] }
 lazy_static = "1.3.0"
-ttrpc = "0.3.0"
+ttrpc = { version = "0.4.13", features=["async", "protobuf-codec"], default-features = false }
```

Comment:
Can you also update `tools/agent-ctl/Cargo.toml` (currently fails to build with this PR).

Reply:
done